app/views/layouts/_alerts.html.erb

```diff
@@ -1,16 +1,30 @@
 <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track' => true %>
 
 <% if notice and alert %>
-	<div style="background-color:gray" class="alerts">
```

Comment:
Will be much more visually pleasing; gray background was a good placeholder, but now on to better things :D